bin/sdk-cli.js

```diff
@@ -73,14 +73,21 @@ const build = ( target, argv ) => {
 	} );
 };
 
+// Generic "watch" option you can add to any command
+const watch = {
+	alias: 'w',
+	description: 'Whether to watch for changes and automatically rebuild.',
+	type: 'boolean',
+};
+
```

Comment:
This appears to come straight from the "gutenberg" version of this command 👍

Reply:
Correct. :-)

Reply:
The reason why I extracted was because it's also "hardcoded" in the `build`:

https://github.com/Automattic/wp-calypso/blob/fbf07c14889404776c912681a5aeefa7e732c527/bin/sdk-cli.js#L54-L55